scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -1638,4 +1638,5 @@ def verifyDBVersion(version):
         update_db("4.0.2")      # Note: version here should be updated for each ISD release
     elif migrate_data_flag == 'true':        
         perform_migration("4.0.2")       # pass the ISD version we are performing data migration for (Note: version here should be updated for each ISD release)
+        audit_conn.autocommit = True
```

Comment:
@Pranav-b-7 why just audit_conn; why not other connections as well which are used in the perform_migration method (I guess, platform_conn, spindb_orca and spindb_front50) ?

Reply:
spindb_orca and spindb_front50 are spinnaker DBs . So we need not worry about that.  platform. i missed it. will add them